### PR TITLE
[CHK-618] Add Handlebars mail templates

### PR DESCRIPTION
**Pull request by dmnplb**
#### List of Changes
This PR adds the following Handlebars mail templates with relative JSON schema (and JSON example):
- Receipt · KO
- Receipt · Success (with different user scenarios)

#### Context
You can find the original source of these templates in the [pagopa-email-template's repository](https://github.com/pagopa/pagopa-email-templates)

#### How Has This Been Tested?
Different user scenarios have been tested during the templates' development. You can check out the different JSON example files in the repository mentioned above.

### CODE REVIEW

**Comment by giovanniberti** [comment]
One minor nitpick: could you rename the template files with the convention from the DR? (i.e. `success.template.html`, `success.template.pdf.html`, etc.)

File: src/templates/success/success.json [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+      },
+      "required": ["email"]
+    },
+    "cart": {
+      "type": "object",
+      "properties": {
+        "items": {
+          "type": "array",
+          "additionalItems": false,
+          "items": [
```
Comment (giovanniberti) [inline src/templates/success/success.json:101]: Typo: `items` element should be an object, not an array

File: src/templates/ko/ko.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
```
Comment (infantesimone) [inline src/templates/ko/ko.json:3]: add   `"$id": "templates/success/schema.json"`

File: src/templates/success/success.json [hunk 3]
```diff
@@ -0,0 +1,176 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "success",
```
Comment (infantesimone) [inline src/templates/success/success.json:3]: add   `"$id": "templates/success/schema.json"`

File: src/templates/success/success.template.html [hunk 4]
```diff
@@ -0,0 +1,836 @@
...
+                          </table>
+                        </div>
+                        <!--[if mso | IE]></td></tr></table><![endif]-->
+                      </td>
+                    </tr>
+                  </tbody>
+                </table>
+              </div>
+              <!--[if mso | IE]></td></tr></table></td></tr><![endif]-->
+              {{#eq cart.items.length 1}}
```
Comment (infantesimone) [inline src/templates/success/success.template.html:344]: To use `eq`, is to be introduced  custom helper or something like https://www.npmjs.com/package/handlebars-helpers

Comment (dmnplb) [reply src/templates/success/success.template.html:344]: @infantesimone That's right. We currently use two custom helpers: `eq` (equivalent to `a === b`) and `not` (equivalent to `a !== b`). We can add these custom helpers ([which you can find in this repo](https://github.com/pagopa/pagopa-email-templates/tree/main/Receipt_Success/helpers)) or directly use [handlebars-helpers](https://www.npmjs.com/package/handlebars-helpers) as you suggested.

File: src/templates/ko/ko.json (hunk 2 above)
Comment (giovanniberti) [reply src/templates/ko/ko.json:4]: Solved with [ff7ecf2](https://github.com/pagopa/pagopa-notifications-service/pull/15/commits/ff7ecf2dd41be737d2862ea9e884634820f0c3e6)

File: src/templates/success/success.json (hunk 3 above)
Comment (giovanniberti) [reply src/templates/success/success.json:4]: Solved with [ff7ecf2](https://github.com/pagopa/pagopa-notifications-service/pull/15/commits/ff7ecf2dd41be737d2862ea9e884634820f0c3e6)

File: src/templates/success/success.template.html (hunk 4 above)
Comment (giovanniberti) [reply src/templates/success/success.template.html:344]: Solved with [42f6d3c](https://github.com/pagopa/pagopa-notifications-service/pull/15/commits/42f6d3ce745c7520ae50ebd47ccf1c089bed3af8)

**Review by infantesimone — APPROVED** [review]

**Pull request merged**